src/components/datetime/editForm/DateTime.edit.date.js

```diff
@@ -11,20 +11,20 @@ export default [
     tooltip: 'Enables date input for this field.'
   },
   {
-    type: 'textfield',
+    type: 'datetime',
```

Comment:
This removes the ability to use moment to add min and max dates like the following.

```js
moment().subtract(10, \'days\')
```

Reply:
@travist 
Updated with simple mask since there is no possibility to add moment object to datetime input.
Hope that's enough.